presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -136,7 +136,7 @@ public static DistributedQueryRunner createQueryRunner(
 
         DistributedQueryRunner queryRunner =
                 DistributedQueryRunner.builder(createSession(Optional.of(new SelectedRole(ROLE, Optional.of("admin")))))
-                        .setNodeCount(workerCount.orElse(4))
+                        .setNodeCount(workerCount.orElse(1))
```

Comment:
Let's leave this part untouched as Hive query runner will be used by a lot of integration tests. We will need parallelism coverage

Reply:
Forgot to put it back?

Reply:
Interesting I remembered having changed it. Maybe somewhere in git operations reverted it back.